test/load/allocation/grpc/runAllocation.sh

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+# The number of times you want allocation test to be run
+TESTRUNSCOUNT=3
+NAMESPACE=default
+EXTERNAL_IP=<IP_ADRESSS_TO_THE_ALLOCATOR_SERVICES_LOAD_BALANCER>
+KEY_FILE=client.key
+CERT_FILE=client.crt
+TLS_CA_FILE=ca.crt
```

Comment:
Let's make this all able to be passed via the command line/set env args, without having to overwrite the script, while also providing some defaults that make life easy.

```suggestion
TESTRUNSCOUNT=${TESTRUNSCOUNT:-3}
NAMESPACE=${NAMESPACE:-default}
EXTERNAL_IP=$(kubectl get services agones-allocator -n agones-system -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
KEY_FILE=${KEY_FILE:-client.key}
CERT_FILE=${CERT_FILE:-client.crt}
TLS_CA_FILE=${TLS_CA_FILE:-ca.crt}
```

Reply:
thanks for the suggestion. Updated